# Hi, I am Geb

I work as Research Assistant in the field of Applied Data Sciences and Statistics in the department of Mathematics in Khalifa University. I am interested in data analytics and data science. I have recently taken and completed the Google Data Analytics Certification Course on coursera. Some of the projects I have worked in are listed below.

## [Project 1: Google Data Analytics Certificate Capstone - Case Study: How does a Bike-share Navigate Speedy Success?](https://github.com/ghiwotkal/Capstone_project/blob/main/Case%20study%20Cyclistic%20Bike-share%20Analysis.ipynb)
The marketing department in Cyclistic, a fictional company, wanted to design a new marketing strategy to maximize the Cyclistic companyâ€™s profit and wanted to know how casual riders and annual members use Cyclistic bikes differently for a speedy success.
### Activities performed in this case study:
* Clearly defined the statement of the business task
* Prepared and described all data 
* Cleaned and processed historical bycicle trip data 
* Analyzed historical bycicle data in order to identify trends, understand how casual riders behave differently from riders with paid memberships using visualizations
* Proposed recommendations to  executives to make decisions about marketing programs and strategies to convert casual riders to riders with annual memberships to increase profit.

![Casual riders vs annual members](https://github.com/ghiwotkal/Capstone_project/blob/main/capstone.png)



## [Project 2: PCA for Dimensionality Reduction](https://github.com/ghiwotkal/plinkPCA/blob/main/R%20Notebook.pdf)
* Principal component analysis (PCA), a dimensionality reduction method, was applied to the Human Genome Diversity Panel (HGDP-CEPH) data to understand the population structure. 
* HGDP data was originally  in a text format and was converted to .ped format and then the .ped formats to .bed, .bim and .fam
* Then plink command line codes were used to produce egenvalues and eigenvectors of the HGDP data.
* plinkPCA was applied to the HGDP data to visualize and understand the population stucture.


![PC1 vs PC2 of the HGDP data](https://github.com/ghiwotkal/plinkPCA/blob/main/PCA%20of%20HGDP%20(PC1%20vs%20PC2).png)


![PC3 vs PC4 of the HGDP data](https://github.com/ghiwotkal/plinkPCA/blob/main/PC3%20vs%20PC4%20of%20HGDP.png)

## [Project 3: covid19 wastewater projectr report generator script](https://github.com/ghiwotkal/covid19_wastewater/blob/main/wastewater_covid19_detection_report.Rmd) 
In here, a small R script that can take an excel sheet or csv file as input and generate a report with  [this format](https://github.com/ghiwotkal/covid19_wastewater/blob/main/wastewater_covid19_detection_report.pdf) was written.

## [Project 4: Using bigsnpr to do  Principal component Analysis (PCA) and Genome Wide Association Stduy(GWAS) of gentic data](https://github.com/ghiwotkal/bigsnprPCA)
* In this project, an R package called `bigsnpr` was used to do principal component analysis and GWAS of a real bmi genetic data. The package reads a plink (bed/bim/fam) file formats.
* The bmi data, which was originally given as .ped and .map, was first converted to .bed, .bim and .fam formats using a command line code.

The first 12 principal components

![The Principal Component Analysis](https://github.com/ghiwotkal/bigsnprPCA/blob/main/PCAs.PNG)


![The quantil-quantile plot](https://github.com/ghiwotkal/bigsnprPCA/blob/main/gwas.PNG)

The manahattan plot

![manhattan plot](https://github.com/ghiwotkal/bigsnprPCA/blob/main/manhattan%20plot.PNG)


## [Project 5: Dimensionality Reduction Methods for Visualizing Population Genetics](https://github.com/ghiwotkal/popviz/blob/main/Dimensionality_Reduction_Methods.pdf)

Over the years, different dimensionality reduction and visualization methods have been implemented in a bid to
visualize the sabtle features of high dimensional data. These methods have their own strengths and shortcomings. In here, a comprehensive review of these methods and their applications in visualizing population
genetics is presented. Performances of the methods are investigated on the hgdp-ceph, 1000 genome and
hapmap3 genomic datasets. It explains the results and suggests which method to use in visualizing
high dimensional genomic data.

You can see a sample preprocessing of the data
https://github.com/<ghiwotkal>/<popviz>/blob/main/Preprocessing_and_PCA_Analysis_of_the_1000G_2504_data.html
